pkg/scheduler/plugins/elasticquota/plugin_helper.go

```diff
@@ -239,12 +243,15 @@ func (g *Plugin) createRootQuotaIfNotPresent() {
 	klog.Infof("create RootQuota successfully")
 }
 
-func (g *Plugin) snapshotPostFilterState(quotaInfo *core.QuotaInfo, state *framework.CycleState) *PostFilterState {
+func (g *Plugin) snapshotPostFilterState(quotaInfo *core.QuotaInfo, state *framework.CycleState, disableRuntimeQuota bool) *PostFilterState {
```

Comment:
It is recommended not to sense disableRuntimeQuota here. It is more appropriate to directly pass in a limit value, and calling it usedLimit is not very appropriate. You can call it limit directly.

Reply:
done